Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/NodeActionListener.java

```diff
@@ -79,8 +79,9 @@ protected abstract void actionPerformed(NodeActionEvent e)
             EventUtil.logEvent(EventType.info, operation, buildProp(node));
             actionPerformed(e);
             return Futures.immediateFuture(null);
-        } catch (AzureCmdException ex) {
+        } catch (AzureCmdException | RuntimeException ex) {
             EventUtil.logError(operation, ErrorType.systemError, ex, null, null);
+            AzureExceptionHandler.getInstance().handleException(ex, false);
             return Futures.immediateFailedFuture(ex);
```

Comment:
what if the owner of the future instance handle the exception again.

Reply:
It was called by [fireNodeActionEvent](https://github.com/microsoft/azure-tools-for-java/blob/develop/Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/NodeAction.java#L69), which only handle the after event listener